cvprac/cvp_api.py

```diff
@@ -61,22 +61,23 @@ class CvpApi(object):
     '''
     # pylint: disable=too-many-public-methods
 
-    def __init__(self, clnt):
+    def __init__(self, clnt, request_timeout=30):
         ''' Initialize the class.
 
             Args:
                 clnt (obj): A CvpClient object
         '''
         self.clnt = clnt
         self.log = clnt.log
+        self.request_timeout = request_timeout
 
     def get_cvp_info(self):
         ''' Returns information about CVP.
 
             Returns:
                 cvp_info (dict): CVP Information
         '''
-        return self.clnt.get('/cvpInfo/getCvpInfo.do')
+        return self.clnt.get('/cvpInfo/getCvpInfo.do', self.request_timeout)
```

Comment:
The timeout parameter is a named parameter so I think the syntax should be:
timeout=self.request_timeout